ddtrace/contrib/logging/patch.py

```diff
@@ -60,20 +61,23 @@ def _w_makeRecord(func, instance, args, kwargs):
     # Get the LogRecord instance for this log
     record = func(*args, **kwargs)
 
-    setattr(record, RECORD_ATTR_VERSION, ddtrace.config.version or "")
-    setattr(record, RECORD_ATTR_ENV, ddtrace.config.env or "")
-    setattr(record, RECORD_ATTR_SERVICE, ddtrace.config.service or "")
+    setattr(record, RECORD_ATTR_VERSION, config.version or "")
+    setattr(record, RECORD_ATTR_ENV, config.env or "")
+    setattr(record, RECORD_ATTR_SERVICE, config.service or "")
 
     # logs from internal logger may explicitly pass the current span to
     # avoid deadlocks in getting the current span while already in locked code.
     span_from_log = getattr(record, _LOG_SPAN_KEY, None)
     if isinstance(span_from_log, ddtrace.Span):
         span = span_from_log
     else:
-        span = _get_current_span(tracer=ddtrace.config.logging.tracer)
+        span = _get_current_span(tracer=config.logging.tracer)
 
     if span:
-        setattr(record, RECORD_ATTR_TRACE_ID, str(span.trace_id))
+        trace_id = span.trace_id
+        if config._128_bit_trace_id_enabled and not config._128_bit_trace_id_logging_enabled:
```

Comment:
I'm wondering if checking the `DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED` variable here is redundant, since if `DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED=False`, won't we by default have a 64-bit trace_ID?

Reply:
I just wanted this code to be explicit. This condition only matters if `DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED=True`. I'll remove it